platforms.cfg

```diff
@@ -177,6 +177,9 @@ pcengine_theme="pcengine"
 pcfx_exts=".pce .ccd .cue .iso .chd"
 pcfx_fullname="PC-FX"
 
+pico8_exts=".p8 .zip"
```

Comment:
is .p8.png not being allowed to get around libretro opening the file as an image?

I know you can disable the imageviewer in retroarch as an override and then it can load .p8.png files just fine

Reply:
@theofficialgman 
Can you explain your experience in the pico-8 thread, so we can test and see if it is possible to implement this later into the module-script ?
https://retropie.org.uk/forum/topic/30965/pico-8-retroarch-lr-retro8-core-installation-script

Reply:
I can but the change is simple... 
all I did was add ```builtin_imageviewer_enable = "false"``` to the pico8 retroarch.cfg (the one that can be made in /opt/retropie/configs/pico8)

Reply:
not exactly sure which command in retropie does this for you but I know the resulting file looks like this
```
# Settings made here will only override settings in the global retroarch.cfg if placed above the #include line



input_remapping_directory = "/opt/retropie/configs/pcengine/"



#include "/opt/retropie/configs/all/retroarch.cfg"
```

Reply:
currently it looks like there is only this helper: ```function ensureSystemretroconfig()``` which can't write arbitrary contents in this file
@joolswills how do you think disabling the imageviewer should be done? or would you rather just have the user rename all of their pico8 files?

Reply:
That function should suffice for new installs. But I've not yet tested so not sure I understand the issue completely.

Reply:
@joolswills I was just saying that the ensureSystemretroconfig() function currently is the only thing in retropie that writes to the /opt/retropie/configs/systemnamehere/retroarch.cfg as part of the configuration part of an install

pico8 files are normally have an ending of .p8.png. retroarch falsely interprets this as a standard image and won't load the core but will instead display the image of the game. the imageviewer needs to be disabled and I was asking how you want to go about writing to the  /opt/retropie/configs/pico8/retroarch.cfg file. I wasn't sure if doing something like this would be allowed in the  scriptmodules/libretrocores/lr-retro8.sh

```
tee /opt/retropie/configs/pico8/retroarch.cfg <<'EOF' >>/dev/null
# Settings made here will only override settings in the global retroarch.cfg if placed above the #include line

builtin_imageviewer_enable = "false"

#include "/opt/retropie/configs/all/retroarch.cfg"
EOF

Reply:
The games are stored in a .PNG file? Oh I see.

Probably would be good if this was sorted upstream as the core should probably be allowed to take priority and load the files over the viewer but you can just do after the ensureSystemretroconfig call
```
iniConfig " = " "" "$configdir/pico8/retroarch.cfg"
iniSet "builtin_imageviewer_enable" "false"
```

Reply:
The ensureSystemretroconfig call is very old and a bit limited. It would be better if it accepted some keys/values or something for defaults as that would be cleaner. The iniSet will always override any user settings which isn't ideal (we prefer to provide details but not force configs unless we have to) whereas the ensureSystemretroconfig call will not.

I'm going to make a new function and maybe deprecate the old one but for now the ini way above is the best fix